pkg/local_object_storage/engine/inhume.go

```diff
@@ -179,7 +179,8 @@ func (e *StorageEngine) inhumeAddr(addr oid.Address, prm shard.InhumePrm, checkE
 	return ok, retErr
 }
 
-func (e *StorageEngine) isLocked(addr oid.Address) (bool, error) {
+// IsLocked checks whether an object is locked according to StorageEngine's state.
+func (e *StorageEngine) IsLocked(addr oid.Address) (bool, error) {
```

Comment:
previously was called under `execIfNotBlocked` condition only, pls double-check do we need to keep this or not

Reply:
fixed